testing/kuttl/e2e/exporter-standby/03--create-standby.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- files/standby-cluster.yaml
+# - files/update-standby-password.yaml
```

Comment:
Are the commented out lines remnants of trying to create + update in one step?

Reply:
I was doing both of these (create and update) in one step at one point. I can remove it